### PR TITLE
Add LockedByID stargate whitelist

### PR DESCRIPTION
**Pull request by iboss-ptk**
## What is the purpose of the change

Add LockedByID stargate whitelist so that CosmWasm contract can query lock

## Brief Changelog

  - Add LockedByID stargate whitelist

## Testing and Verifying

This change is a trivial rework / code cleanup without any test coverage.

## Documentation and Release Note
  - Does this pull request introduce a new feature or user-facing behavior changes? (no)
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? (no)
  - How is the feature or change documented? (not applicable)

### CODE REVIEW

File: x/lockup/keeper/msg_server.go [hunk 1]
```diff
@@ -183,28 +183,31 @@ func (server msgServer) ForceUnlock(goCtx context.Context, msg *types.MsgForceUn
...
+	// for _, addr := range forceUnlockAllowedAddresses {
+	// 	// defense in depth, double checking the message owner and lock owner are both the same and is one of the allowed force unlock addresses
+	// 	if addr == lock.Owner && addr == msg.Owner {
+	// 		found = true
+	// 		break
+	// 	}
+	// }
+	// if !found {
+	// 	return &types.MsgForceUnlockResponse{Success: false}, sdkerrors.Wrapf(sdkerrors.ErrUnauthorized, "Sender (%s) not allowed to force unlock", lock.Owner)
+	// }
```
Comment (mattverse) [inline x/lockup/keeper/msg_server.go:211]: need to update these 😄

Comment (iboss-ptk) [reply x/lockup/keeper/msg_server.go:211]: oopss

Comment (iboss-ptk) [reply x/lockup/keeper/msg_server.go]: reverted

**Review by p0mvn — APPROVED** [review]
LGTM once CI is passing.

Confirmed that the query is both deterministic and constant in gas usage.

**Pull request merged**